tests/ed25519_test.c

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include <aws/cal/ed25519.h>
+
+#include <aws/testing/aws_test_harness.h>
+
+#include "test_case_helper.h"
+
+static int s_ed25519_key_pair_generate_test(struct aws_allocator *allocator, void *ctx) {
```

Comment:
Is there a way to verify the generated values instead of just the length?

Reply:
not in an easy way. its either just reconstructing the format and ignoring parts that change or mocking lc pieces to generate the same keys.